README.md

```diff
@@ -43,7 +43,6 @@ order to force the meeting to show up both on the front page as well as
 in the meetings tab:
 
 * **save_as** - Set this to *index.html*
-* **url** - Leave this blank
```

Comment:
@bdbaddog I think I made a mistake. I thought I needed to remove this because of our recent issues with browsing past meetings. HOWEVER, I am now starting to understand that this is here on purpose for the `index.html` versions *AND* it should be removed on past versions when we have a new `index.html`.

If what i said above is correct, I need to revert this commit: `f9144c84b8a695fe862ead6e15eb812e26bac767`

@bdbaddog Please advice/confirm for me.

Reply:
Yes. save_as: index.html is meant to be the next meeting and then removed from that and changed to the next one when it's past.